src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="/home">
```

Comment:
For this purpose I would rather use links with hashtags, as navigation links in most cases will point to the specific sections on the page
```suggestion
      <a class="header__logo" href="#home">
```